src/electron/index.ts

```diff
@@ -14,71 +14,74 @@
 
 // Directly import @sentry/electron main process code.
 // See: https://docs.sentry.io/platforms/javascript/guides/electron/#webpack-configuration
-import * as sentry from '@sentry/electron/dist/main';
-import {app, BrowserWindow, ipcMain, Menu, MenuItemConstructorOptions, nativeImage, shell, Tray} from 'electron';
-import * as promiseIpc from 'electron-promise-ipc';
-import {autoUpdater} from 'electron-updater';
-import * as os from 'os';
-import * as path from 'path';
-import * as process from 'process';
-import * as url from 'url';
-import autoLaunch = require('auto-launch'); // tslint:disable-line
-
-import * as connectivity from './connectivity';
-import * as errors from '../www/model/errors';
-
-import {ShadowsocksConfig} from '../www/app/config';
-import {TunnelStatus} from '../www/app/tunnel';
-import {GoVpnTunnel} from './go_vpn_tunnel';
-import {RoutingDaemon} from './routing_service';
-import {ShadowsocksLibevBadvpnTunnel} from './sslibev_badvpn_tunnel';
-import {TunnelStore, SerializableTunnel} from './tunnel_store';
-import {VpnTunnel} from './vpn_tunnel';
+import * as sentry from "@sentry/electron";
+import {app, BrowserWindow, ipcMain, Menu, MenuItemConstructorOptions, nativeImage, shell, Tray} from "electron";
+import promiseIpc from "electron-promise-ipc";
```

Comment:
Fix `TypeError: promiseIpc.on is not a function`, which is also [documented here](https://www.npmjs.com/package/electron-promise-ipc).